docs/hugo/content/guide/asov1-asov2-migration/_index.md

```diff
@@ -164,11 +181,13 @@ ASOv1 exports secrets from Azure according to the
 rules. ASOv1 always exports these secrets, there is no user configuration required on either the name of the secret or its values.
 In contrast, ASOv2 requires the user to opt-in to secret export, via the `spec.operatorSpec.secrets` property.
 
-> ℹ️ **Note**: ASOv2 also splits secrets/configuration into two categories: Secrets provided by you, and secrets generated 
-> by Azure. Exporting secrets from Azure is done via the `spec.operatorSpec.secrets`, while supplying secrets to Azure
-> is done by passing the reference to a secret key/value, such as via the 
-> [administratorLoginPassword field of Azure SQL](../../reference/sql/v1api20211101#sql.azure.com/v1api20211101.Server_Spec).
-> This means that there may be cases where a single secret in ASOv1 becomes 2 secrets in ASOv2, one for inputs and one for outputs.
+{{% alert title="Note" %}}
+ASOv2 also splits secrets/configuration into two categories: Secrets provided by you, and secrets generated
```

Comment:
You mention configuration up front, but then don't mention it at all.

Either document config as well, or remove it altogether.

Reply:
Dropped it as ASOv1 doesn't do ConfigMaps and so from a migration perspective only secrets is interesting here. ConfigMap export is a new feature that they can use in ASOv2 but the samples and other docs can help them discover that, focus here should be on migration.